integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseFileSystemOpts.java

```diff
@@ -67,10 +66,8 @@ public static AlluxioFuseFileSystemOpts create(AlluxioConfiguration conf) {
    */
   public static AlluxioFuseFileSystemOpts create(
       AlluxioConfiguration conf, AlluxioFuseCliOpts fuseCliOpts) {
-    Preconditions.checkNotNull(conf,
-        "AlluxioConfiguration should not be null for creating AlluxioFuseFileSystemOpts.");
-    Preconditions.checkNotNull(fuseCliOpts,
-        "AlluxioFuseCliOpts should not be null for creating AlluxioFuseFileSystemOpts.");
+    Preconditions.checkNotNull(conf);
```

Comment:
Most of the checkNotNull objects are created internally and not seeable by end users, so there's no need to add more messages. The ones that are related to users we don't know the exact runtime exception, so we can't know more information now, and thus no need of messages